paths/entities/items/labels/singleLang.json

```diff
@@ -11,5 +11,26 @@
             "404": { "$ref": "../../../../responses.json#/NotFound" },
             "default": { "$ref": "../../../../responses.json#/UnexpectedError" }
         }
+    },
+    "put": {
+        "tags": ["items"],
+        "summary": "Add / Replace a label to an entity by language",
+        "parameters": [
+            { "$ref": "../../../../parameters.json#/entityId" },
+            { "$ref": "../../../../parameters.json#/lang" }
+        ],
+        "requestBody": { "$ref": "../../../../requests.json#/Item" },
+        "responses": {
+            "200": { 
+                "description": "Wikibase Entity ID",
+                "content": {
+                    "application/json": {
+                        "schema": { "type": "string" }
+                    }
+                }
+             },
```

Comment:
The response should probably be the fingerprint itself, in lack of a proper ID: `EntityLabel`